components/alert-dialog/src/tokens.json

```diff
@@ -1,69 +1,105 @@
 {
   "utrecht": {
     "alert-dialog": {
+      "margin-block-start": {
+        "$extensions": {
+          "nl.nldesignsystem.css.property": {
+            "syntax": "<length>",
+            "inherits": true
+          },
+          "nl.nldesignsystem.figma.supports-token": false
+        },
+        "type": "spacing"
+      },
+      "margin-block-end": {
+        "$extensions": {
+          "nl.nldesignsystem.css.property": {
+            "syntax": "<length>",
+            "inherits": true
+          },
+          "nl.nldesignsystem.figma.supports-token": false
+        },
+        "type": "spacing"
+      },
```

Comment:
Is het logisch dat een dialog margin heeft? Als de CSS dit heeft, moeten we dit misschien uit de CSS verwijderen. @jeffreylauwers Wat denk jij?

Reply:
interesting thought! we'll keep it for now

We moeten dit wel nog op die manier implementeren in CSS